core/templates/dev/head/pages/library/Library.js

```diff
@@ -80,6 +80,31 @@ oppia.controller('Library', [
       $http.get('/libraryindexhandler').success(function(data) {
         $scope.libraryGroups = data.activity_summary_dicts_by_category;
 
+        $http.get('/creatordashboardhandler/data')
+          .then(function(response) {
+            if (response.data.explorations_list !== undefined) {
+              for (var i = 0; i < $scope.libraryGroups.length; i++) {
+                for (var j = 0; j < $scope.libraryGroups[i]
```

Comment:
$scope.libraryGroups[i].activity_summary_dicts.length is clumsy here; make a new variable if you need to for readability.

Reply:
okay